tests/test_text.out

```diff
@@ -0,0 +1,34 @@
+# field1,field2,field3
```

Comment:
Was this mistakenly committed?

Reply:
Yes. My bad. Will delete it. Sorry about that!